src/main/java/com/websecure/wsapi/Controller/DappController.java

```diff
@@ -0,0 +1,34 @@
+package com.websecure.wsapi.Controller;
+
+import com.websecure.wsapi.Document.Dapp;
+import com.websecure.wsapi.Repository.DappRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RequestMapping("/dapps")
+@RestController
+public class DappController {
+
+    @Autowired
+    private DappRepository dappRepository;
+
+    @PostMapping("/{name}")
+    private String addDapp(@RequestBody Dapp dapp){
+        dappRepository.save(dapp);
+        return "Added dapp " + dapp.getId();
+    }
+
+    @GetMapping()
+    private ResponseEntity<List<Dapp>> getDapps(){
+        return new ResponseEntity<>(dappRepository.findAll(), HttpStatus.OK);
+    }
+
+    @GetMapping("/{name}")
```

Comment:
not at all, seen it already

Reply:
I can see there's no delete method,

Reply:
Yeah, do you want to do it?

Reply:
Great, I've seen the changes. Please could you also add the update for dapp, I'm not sure I did that.

Reply:
Done

Reply:
i'll check.

i'm also thinking if we can add an actuator dependency config from spring to monitor the health of the app and it will also give us more endpoints to add the app info (which includes the name, version and also the purpose

Reply:
> Great, I've seen the changes. Please could you also add the update for dapp, I'm not sure I did that.

I just checked, you added it

Reply:
Yes, we can add the actuator, please do. I made some changes, I felt it might be better to return ResponseEntity rather than just String, please look through and I think we can merge, then move to other things.

Reply:
Also when you're done reviewing, I want to squash the commits (merge them into one), so the history looks clean.

Reply:
> Yes, we can add the actuator, please do. I made some changes, I felt it might be better to return ResponseEntity rather than just String, please look through and I think we can merge, then move to other things.

yes it's better, it makes it look more professional.